src/app/store/effects/upload.effects.spec.ts

```diff
@@ -210,4 +210,15 @@ describe('UploadEffects', () => {
       expect(contentManagementService.versionUpdateDialog).toHaveBeenCalledWith(fakeEvent.detail.data.node.entry, fakeEvent.detail.files[0].file);
     });
   });
+
+  describe('image versioning', () => {
+    it('should trigger upload file version from viewer', () => {
```

Comment:
please make sure we test the error for this.contentService.getNodeInfo()

Reply:
tested